lnst/Recipes/ENRT/GeneveOvsTunnelRecipe.py

```diff
@@ -117,7 +120,7 @@ def create_tunnel(self, configuration):
             host.br0.up()
             host.int0.up()
 
-        self.wait_tentative_ips(configuration.tunnel_devices)
```

Comment:
You removed this from the code, so we will now won't wait for tentative flag.

Reply:
Accidentally removed, I returned it back there.